libraries/protocol/steem_operations.cpp

```diff
@@ -188,6 +188,17 @@ namespace steemit { namespace protocol {
    void witness_update_operation::validate() const
    {
       validate_account_name( owner );
+
+      if ( has_hardfork )
```

Comment:
I have to make sure:


1) I only remove below condition from 'witness_update_evaluator::do_apply' method

	if ( _db.has_hardfork( STEEMIT_HARDFORK_0_1 ) )
	{
		FC_ASSERT( o.url.size() <= STEEMIT_MAX_WITNESS_URL_LENGTH, "URL is too long" );
	}

2) I have to add below code to 'witness_update_operation::validate' method.

FC_ASSERT( o.url.size() <= STEEMIT_MAX_WITNESS_URL_LENGTH, "URL is too long" );

You said 'we can safely put the post hardfork logic in the validator without a hardfork check.', so
So, I will add only 'FC_ASSERT', without condition.

Is it correct?